### DISCUSSION TITLE
SEQ Upgrade Question: JSON View Behavior in 2024.4

### DISCUSSION THREAD

**Discussion by kejdajar** [datalust/seq-tickets#2336] [Help] [answered]
Hello,

We are considering upgrading from SEQ 2023.4 to 2024.4. During my testing, I noticed that in the latest version, our logged request body data is displayed as expanded records by default. This causes visual clutter when first opening logged events, especially for POST requests with large request bodies.

Is there a way to set the previous "inlined" JSON view as the default when opening an event?

Thank you!

![seq](https://github.com/user-attachments/assets/a44ee16c-489a-4c8e-af29-757c16fffdc0)

**Comment by liammclennan** [comment, marked as answer]
Hi @kejdajar 

It's called 'Flatten the event properties view' in your preferences.

**Reply by kejdajar** [reply]
Thank you, that’s exactly what I was looking for. Have a nice day!
